css/style.css

```diff
@@ -1116,6 +1116,26 @@ h2 a {
   font-size: 16px;
 }
 
+/* h2 -> h3 header links */
+.heading-link::after {
+  content: '#';
+  color: var(--card-fg);
+}
+
+.heading-link {
+  margin-left: 0.3em;

```

Comment:
```suggestion
  padding-inline: 0.3em;
```

I prefer padding for this, as it expands the clickable area to include the spacing, vs margin which adds spacing that is not clickable

Reply:
I think if we do font size inherit, padding will grow even bigger links especially on h2 headers.

Reply:
This looks good 👍
<img width="235" height="56" alt="image" src="https://github.com/user-attachments/assets/c387f19b-1461-4f24-ad1c-ffd4f324831d" />